[Bug]: Semantic search workload fails while registering ML model

### Describe the bug

```
opensearch-benchmark execute-test \
 --workload-path="/home/ec2-user/.benchmark/benchmarks/workloads/private/trec_covid_semantic_search" \
 --workload-params="/home/ec2-user/.benchmark/benchmarks/workloads/private/trec_covid_semantic_search/params/params.json" \
 --pipeline=benchmark-only \
 --target-host="$EP" \
 --kill-running-processes \
--client-options=timeout:120,amazon_aws_log_in:environment
```

params.json:
```
{
    "bulk_indexing_clients": 4,
    "bulk_size": 200,
    "number_of_replicas": 1,
    "number_of_shards" :8,
    "ingest_percentage":100,
    "search_clients": 8,
    "warmup_iterations": 20,
    "iterations": 100,
    "variable_queries": 50,
    "k": 100,
    "model_name": "text-embedding-model",
    "model_format": "TORCH_SCRIPT",
    "model_version" : "1.0.1",
    "model_config_file" : "/home/ec2-user/.benchmark/benchmarks/workloads/private/trec_covid_semantic_search/model_config_file.json"
}
```

Error:
```
Running delete-index                                                           [100% done]
Running set-concurrent-segment-search                                          [100% done]
Running delete-ingest-pipeline                                                 [100% done]
Running delete-ml-model                                                        [100% done]
Running register-ml-model                                                      [  0% done]
[ERROR] Cannot execute-test. Error in load generator [0]
        Cannot run task [register-ml-model]: Port could not be cast to integer value as '443_plugins'
```

--- a/osbenchmark/worker_coordinator/runner.py
+++ b/osbenchmark/worker_coordinator/runner.py
@@ -2699,7 +2699,7 @@ class DeleteMlModel(Runner):
     @time_func
     async def __call__(self, opensearch, params):
         async def _is_deployed(model_id):
-            resp = await opensearch.transport.perform_request('GET', '_plugins/_ml/models/' + model_id)
+            resp = await opensearch.transport.perform_request('GET', '/_plugins/_ml/models/' + model_id)
             state = resp.get('model_state')
             return state in ('PARTIALLY_DEPLOYED', 'DEPLOYED')
 
@@ -2770,14 +2770,14 @@ class RegisterMlModel(Runner):
                     break
 
         if not model_id:
-            resp = await opensearch.transport.perform_request('POST', '_plugins/_ml/models/_register', body=body)
+            resp = await opensearch.transport.perform_request('POST', '/_plugins/_ml/models/_register', body=body)
             task_id = resp.get('task_id')
             timeout = params.get('timeout', 120)
             end = time.time() + timeout
             state = 'CREATED'
             while state == 'CREATED' and time.time() < end:
                 await asyncio.sleep(5)
-                resp = await opensearch.transport.perform_request('GET', '_plugins/_ml/tasks/' + task_id)
+                resp = await opensearch.transport.perform_request('GET', '/_plugins/_ml/tasks/' + task_id)
                 state = resp.get('state')
             if state == 'FAILED':
                 raise exceptions.BenchmarkError("Failed to register ml-model. Error: {}".format(resp['error']))
@@ -2799,14 +2799,14 @@ class DeployMlModel(Runner):
             d = json.loads(f.read())
             model_id = d['model_id']
 
-        resp = await opensearch.transport.perform_request('POST', '_plugins/_ml/models/' + model_id + '/_deploy')
+        resp = await opensearch.transport.perform_request('POST', '/_plugins/_ml/models/' + model_id + '/_deploy')
         task_id = resp.get('task_id')
         timeout = params.get('timeout', 120)
         end = time.time() + timeout
         state = 'RUNNING'
         while state == 'RUNNING' and time.time() < end:
             await asyncio.sleep(5)
-            resp = await opensearch.transport.perform_request('GET', '_plugins/_ml/tasks/' + task_id)
+            resp = await opensearch.transport.perform_request('GET', '/_plugins/_ml/tasks/' + task_id)
             state = resp.get('state')
         if state == 'FAILED':
             raise exceptions.BenchmarkError("Failed to deploy ml-model. Error: {}".format(resp['error']))

